client/db/bolt/db_test.go

```diff
@@ -224,6 +224,70 @@ func TestAccounts(t *testing.T) {
 	}
 }
 
+func TestDeleteAccount(t *testing.T) {
```

Comment:
This test is covered in `TestDisableAccount`. Can delete, I think.

Reply:
Deleted, with the same reasoning for it's creation as the comment above.